python_on_whales/__init__.py

```diff
@@ -21,7 +21,7 @@
 from .exceptions import DockerException
 
 # alias
-docker = DockerClient()
+docker = DockerClient(client_type="docker")
```

Comment:
If this is because `client_call` defaults to `["docker"]` then why not have this default be set in `DockerClient__init__()`? Your point about the ctr exe possibly being a symlink to something else still stands such that this may be incorrect (unless it actually gets to the point where POW downloads the client).

Reply:
That's true, but then the "unknown" option would never be used, and it has a purpose. It means "as a user I didn't think about it". When people do `from python_on_whales import docker`, we can reasonably assume that they really mean docker, even if you're right, a symlink would be annying here. 

I try to follow the motto "easy things should be easy, hard things should be possible". If there is no "unknown", then it's impossible for the library to tell you that it's assuming incorrectly the type of client.

Reply:
Yeah, that's fine, using `python_on_whales.docker` just comes with the caveat of assuming if you have `docker` on your system it actually is a docker client, I don't have an issue with that.